### ISSUE TITLE
Package version incompatibilities

### ISSUE THREAD

**Issue by PaulZavadski** [m1guelpf/laravel-fastlogin#30]
The package is not compatible with laravel 9 and php version 8.1
Illuminate 8 version needs to be updated to the latest

```composer.json
{
    "name": "m1guelpf/laravel-fastlogin",
    "description": "Allow your users to login with FaceID/TouchID",
    "keywords": [
        "m1guelpf",
        "laravel-fastlogin"
    ],
    "homepage": "https://github.com/m1guelpf/laravel-fastlogin",
    "license": "MIT",
    "authors": [
        {
            "name": "Miguel Piedrafita",
            "email": "soy@miguelpiedrafita.com",
            "role": "Developer",
            "homepage": "https://miguelpiedrafita.com"
        }
    ],
    "require": {
        "php": "^8.0|^8.1",
        "illuminate/contracts": "^9.31",
        "illuminate/database": "^9.31",
        "illuminate/http": "^9.31",
        "illuminate/support": "^9.31",
        "illuminate/validation": "^9.31",
        "nyholm/psr7": "*",
        "psr/http-message": "*",
        "symfony/psr-http-message-bridge": "*",
        "web-auth/webauthn-lib": "*"
    },
    "autoload": {
        "psr-4": {
            "M1guelpf\\FastLogin\\": "src"
        }
    },
    "scripts": {
        "psalm": "vendor/bin/psalm",
        "cs": "php-cs-fixer fix . --config=.php_cs --allow-risky=yes"
    },
    "config": {
        "sort-packages": true
    },
    "extra": {
        "laravel": {
            "providers": [
                "M1guelpf\\FastLogin\\FastLoginServiceProvider"
            ]
        }
    },
    "minimum-stability": "dev",
    "prefer-stable": true
}
```

**Comment by niladam** [comment]
I can see there already is a pull request that tries to fix this - #29 - maybe @m1guelpf can have a look at that and maybe close this ?

**Comment by m1guelpf** [comment]
Closed in #29
